packages/openneuro-app/src/scripts/components/modality-cube/ModalityHexagon.tsx

```diff
@@ -21,7 +23,7 @@ export const ModalityHexagon: FC<ModalityHexagonProps> = ({
     : <i className="fa fa-circle-o-notch fa-spin"></i>
 
   return (
-    <div className="hexagon-wrapper">
+    <div className={`hexagon-wrapper ${size}`}>
```

Comment:
If this can be undefined, it should map that to a string `hexagon-wrapper ` or `hexagon-wrapper` instead of `hexagon-wrapper undefined`. Caught by the snapshot test.